src/map.js

```diff
@@ -1482,7 +1482,9 @@ var map = function (arg) {
       if (m_discreteZoom) {
         m_this.zoom(Math.round(m_this.zoom()));
       }
-      m_this.interactor().options({discreteZoom: m_discreteZoom});
+      if (m_this.interactor()) {
```

Comment:
can we have a situation where we do not have interactor?

Reply:
Yes, if you want to display a static map.  We have a test without an interactor.

Reply:
thanks!